### PR TITLE
example Rmd to start class

### PR DESCRIPTION
**Pull request by lindsaycarr**
First draft of the example Rmd we will walk through at the beginning of an introduction to R class. I intend for people to download this from the `data/` folder just like they grab csv files. Then we can open it, and be able to knit and walk through each line together.

I took the earthquake blog @ldecicco-USGS put together and adapted it so that it used stuff similar to what we will show in class and eliminated the spatial transform stuff (I just used `maps`). People can go to the real blog to see that, but I wanted to mimic more what we will cover in the class. If we decide to teach `ggplot2` in that particular class, we can just have people look at the plotting functions in the actual blog post.

For the figure, that last plot looks a little funky to me...did I do something wrong? It doesn't seem like there's an appropriate blip in the data like the others.

image

### CODE REVIEW

**Comment by ldecicco-USGS** [comment]
It looks like they took that 1 point out, here's how it looks in the blog:

image

That's the trickiness of provisional data I guess....the scientists behind the scenes fix issues making the data a bit of a moving target. I'm guessing when it turns into "accepted" it will also have an "E" or some other comment at that point in time.

I'd just take out that site for this example. Hopefully others don't get adjusted out...it's cool!

**Comment by lindsaycarr** [comment]
Ahhh good point. I actually am just going to make a note about that since others still might change in the future from the blog.

File: content/intro-curriculum/data/data_workflow.Rmd [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+south_to_north <- arrange(unique_sites, desc(dec_lat_va))
+
+# subset data so it only includes September 8th, 2017
+startDateTime <- as.POSIXct("2017-09-07 00:00:00", tz="UTC")
+endDateTime <- as.POSIXct("2017-09-09 00:00:00", tz="UTC")
+gw_data_sep8 <- gw_data %>%
+  filter(dateTime > startDateTime,
+         dateTime < endDateTime) 
+
+# use the dataRetrieval fxn to get more user-friendly column names
```
Comment (aappling-usgs) [inline content/intro-curriculum/data/data_workflow.Rmd]: could spell out `function` to be super-uber noob-friendly

Comment (lindsaycarr) [reply content/intro-curriculum/data/data_workflow.Rmd]: great suggestion! Fixed!

**Comment by lindsaycarr** [comment]
Merging based on 3/4 approvals.

**Pull request merged**